AppKit/CPView.j

```diff
@@ -252,6 +252,8 @@ var CPViewHighDPIDrawingEnabled = YES;
 
     id                  _animator;
     CPDictionary        _animationsDictionary;
+    BOOL                _inhibitDOMUpdates      @accessors(setter=_setInhibitDOMUpdates);
+    BOOL                _forceDOMUpdates        @accessors(setter=_setForceDOMUpdates);
```

Comment:
This flag forces everything, not just DOM updates. Should be renamed to reflect that _forceUpdates or just _force for example.